package.json

```diff
@@ -50,8 +50,8 @@
   ],
   "scripts": {
     "test": "c8 --reporter=html --reporter=text mocha \"test/*/_index.js\" \"**/*.test.js\" --ignore=\"node_modules/**\"",
-    "lint": "eslint . && prettier --list-different \"**/*.js\" --ignore-path .gitignore",
-    "fix": "eslint . --fix && prettier --write \"**/*.js\" --ignore-path .gitignore",
+    "lint": "eslint --ignore-path .gitignore . && prettier --list-different \"**/*.js\" --ignore-path .gitignore",
+    "fix": "eslint --ignore-path .gitignore --fix . && prettier --write \"**/*.js\" --ignore-path .gitignore",
```

Comment:
I used "ignorePatterns" in eslint config. So not all.

Reply:
I'd say better remove ignorePatterns then and just use .gitignore. It's the right file and includes everything that needs to be ignored.

Reply:
Ok, you can remove